client/src/App.js

```diff
@@ -156,79 +156,29 @@ class Document extends React.Component {
         <div className="main">
           <div className="sidebar">SIDE BAR</div>
           <div className="content">
-            <DocumentFromRecipe document={document} />
+            <RenderHTMLElementDocument document={document} />
```

Comment:
We still have the problem with the keyword `document` but we can worry about that later. We have other name issues to worry about too.